lib/src/functions/map.dart

```diff
@@ -86,6 +94,19 @@ final _deepMerge = _function("deep-merge", r"$map1, $map2", (arguments) {
   return _deepMergeImpl(map1, map2);
 });
 
+final _deepRemove =
+    _function("deep-remove", r"$map, $key, $keys...", (arguments) {
+  var map = arguments[0].assertMap("map");
+  var keys = [arguments[1], ...arguments[2].asList];
```

Comment:
According to the spec, this should throw an error for fewer than 2 keys.

Reply:
Good catch.

@nex3: Is this the behavior we actually want? Since the arguments to `deep-remove` are supposed to parallel `get` and `has-key`, would it make sense to allow fewer than 2 keys (either requiring 1 or requiring 0)? If we do require a minimum number of keys, would it make sense to have the signature list them separately? (e.g. `$map, $key1, $key2, $keys...` or something like that)

Reply:
FWIW this is the error message for `map.set()` when `$args` has too few items: https://github.com/sass/dart-sass/pull/1083/files#diff-132af1066cf93730a37dc60a5fb3644fR69

Reply:
I don't think requiring 0 keys makes sense, but only requiring 1 probably does so people can universally use `deep-replace()` instead of `replace()` if they want. You also make a good point about the `$map, $key, $keys...` signature being more in line with `get()`. Can you send a PR to update the spec?